conduce/api.py

```diff
@@ -245,7 +245,11 @@ def wait_for_job(job_id, **kwargs):
     """
     initial_status_countdown = 9
     server_error_count = 0
-    timeout = kwargs.get('timeout', 300) * 2
+
```

Comment:
This might be a bad sign.  I probably assume that `get` returns the default value if the kwarg is `None` in other places too.